ecc-hol/workshops/sandbox/manifest.json

```diff
@@ -1,48 +1,68 @@
-            "filename": "../../introduction/introduction.md"
-        },
-        {
-            "title": "Lab 1: Provision an Instance",
-            "description": "Labs that follow the introduction are numbered, starting with Lab 1",
-            "filename": "../../provision/provision.md"
-        },
-        {
-            "title": "Lab 2: Setup the Environment",
-            "filename": "../../setup/setup.md"
-        },
-        {
-            "title": "Lab 3: Load Data into an Instance",
-            "filename": "../../data-load/data-load.md"
-        },
-        {
-            "title": "Lab 4: Query Data",
-            "filename": "../../query/query.md"
-        },
-        {
-            "title": "Lab 6: Variables in LiveLabs",
-            "filename": "../../variables/variables.md"
-        },
-        {
-            "title": "Lab 5: Tables in LiveLabs",
-            "filename": "../../tables/tables.md"
-        },
-        {
-            "title": "Need Help?",
-            "description": "Solutions to Common Problems and Directions for Receiving Live Help",
-            "filename":"https://oracle-livelabs.github.io/common/labs/need-help/need-help-livelabs.md"
-        }
+      {
+        "title": "Get started",
+        "description": "The Introduction is always second for LiveLabs. The title and contents menu title match for the Introduction.",
+        "filename": "../../query/Getstarted.md"
+      }, 
+    
+      
+      {
+        "title": "Lab 1: Create ECC schema",
+        "filename": "../../query/create-ecc-schema.md"
+      },
+      {
+        "title": "Lab 2: Create ECC domain",
+        "filename": "../../query/create-ecc-domain.md"
+      },
+      {
+        "title": "Lab 3: Register ECC instance",
+        "filename": "../../query/register-ecc-instance.md"
+      },
+      {
+        "title": "Lab 4: Import ECC applications",
+        "filename": "../../query/import-ecc.md"
+      },
+      {
+              "title": "Lab 5: Run dataload",
+              "filename": "../../query/run-dataload.md"
+      },
+      {
+        "title": "Lab 6: ECC anatomy",
+        "filename": "../../query/ecc-anatomy.md"
+      },
+  
+  
+  {
+    "title": "Lab 7: Power user personalization",
+    "filename": "../../query/poweruser-personalization.md"
+  },
+  {
+    "title": "Lab 8: Admin user personalization",
+    "filename": "../../query/adminuser-personalization.md"
+  },
+  
+  {
+    "title": "Lab 9: Diagnose and monitor",
+    "filename": "../../query/diagnose-and-monitor.md"
+  },
+  {
+    "title": "Additional optional labs: Start and stop ECC",
+    "filename": "../../query/start-stop-ecc.md"
+  },
+  {
+    "title": "Additional optional labs: Improve performance",
+    "filename": "../../query/improve-performance.md"
+  },
+      {
+        "title": "Need help?",
+        "description": "Solutions to Common Problems and Directions for Receiving Live Help",
+        "filename": "../../query/needhelp.md"
```

Comment:
@ecchol Please use the common labs need help lab -

Desktop/Sandbox version - https://oracle-livelabs.github.io/common//labs/need-help/need-help-livelabs.md
Tenancy version - https://oracle-livelabs.github.io/common//labs/need-help/need-help-freetier.md

Reply:
updated the file as per comments. Thank you